packages/sui-decorators/src/decorators/cache/index.js

```diff
@@ -20,11 +20,12 @@ const _cache = ({
   size,
   target,
   ttl,
-  redis
+  redis,
+  cacheKeyString
 } = {}) => {
   const shouldUseRedisCache = redis && isNode
   const shouldUseMemoryCache = !isNode || !redis
-  const cacheKey = `${target.constructor.name}::${fnName}`
+  const cacheKey = cacheKeyString ?? `${target.constructor.name}::${fnName}`
```

Comment:
INMHO it is safer to use falsy check (`?`) than using nullish coalescing operator (`??`) in this case.